dataframe-arrow/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/ArrowKtTest.kt

```diff
@@ -459,4 +471,86 @@ internal class ArrowKtTest {
+            LocalDateTime.of(2015, 5, 25, 14, 20, 13),
+            LocalDateTime.of(2013, 6, 19, 11, 20, 13)
+        )
+
+        val dataFrame = dataFrameOf(
+            "ts_nano" to dates,
+            "ts_micro" to dates,
+            "ts_milli" to dates,
+            "ts_sec" to dates
+        )
+
+        DataFrame.readArrowFeather(writeArrowTimestamp(dates)) shouldBe dataFrame
+        DataFrame.readArrowIPC(writeArrowTimestamp(dates, true)) shouldBe dataFrame
+    }
+
+    private fun writeArrowTimestamp(dates: List<LocalDateTime>, streaming: Boolean = false): ByteArray {
+        RootAllocator().use { allocator ->
+            val timeStampMilli = Field(
+                "ts_milli",
+                FieldType.nullable(ArrowType.Timestamp(TimeUnit.MILLISECOND, null)),
+                null
+            )
+
+            val timeStampMicro = Field(
+                "ts_micro",
+                FieldType.nullable(ArrowType.Timestamp(TimeUnit.MICROSECOND, null)),
+                null
+            )
+
+            val timeStampNano = Field(
+                "ts_nano",
+                FieldType.nullable(ArrowType.Timestamp(TimeUnit.NANOSECOND, null)),
+                null
+            )
+
+            val timeStampSec = Field(
+                "ts_sec",
+                FieldType.nullable(ArrowType.Timestamp(TimeUnit.SECOND, null)),
+                null
+            )
+            val schemaTimeStamp = Schema(
+                listOf(timeStampNano, timeStampMicro, timeStampMilli, timeStampSec)
+            )
+            VectorSchemaRoot.create(schemaTimeStamp, allocator).use { vectorSchemaRoot ->
+                val timeStampMilliVector = vectorSchemaRoot.getVector("ts_milli") as TimeStampMilliVector
+                val timeStampNanoVector = vectorSchemaRoot.getVector("ts_nano") as TimeStampNanoVector
+                val timeStampMicroVector = vectorSchemaRoot.getVector("ts_micro") as TimeStampMicroVector
+                val timeStampSecVector = vectorSchemaRoot.getVector("ts_sec") as TimeStampSecVector
+                timeStampMilliVector.allocateNew(dates.size)
+                timeStampNanoVector.allocateNew(dates.size)
+                timeStampMicroVector.allocateNew(dates.size)
+                timeStampSecVector.allocateNew(dates.size)
+
+                dates.forEachIndexed { index, localDateTime ->
+                    val instant = localDateTime.toInstant(ZoneOffset.UTC)
+                    timeStampNanoVector[index] = instant.toEpochMilli() * 1_000_000L + instant.nano
+                    timeStampMicroVector[index] = instant.toEpochMilli() * 1_000L
+                    timeStampMilliVector[index] = instant.toEpochMilli()
+                    timeStampSecVector[index] = instant.toEpochMilli() / 1_000L
+                }
+                vectorSchemaRoot.setRowCount(dates.size)
+                val bos = ByteArrayOutputStream()
+                bos.use { out ->
+                    val arrowWriter = if (streaming){
```

Comment:
not sure how the linter missed this, but there's supposed to be a space in between `){`